taskcluster/darwin-opt-base.tyml

```diff
@@ -35,11 +35,8 @@ payload:
   # There is no VM yet running tasks on OSX
   # so one should install by hand:
   # - brew
-  # - xcode (brew would install)
-  # - brew install gnu-tar
-  # - brew install git
-  # - brew install pixz
-  # - brew cask install java
+  # - Xcode 10.1 in /Applications/Xcode.app, then sudo chown -R root:wheel /Applications/Xcode.app
+  # - brew install gnu-tar git pixz wget coreutils pyenv-virtualenv
```

Comment:
nit: now we have build/cache support, I'm wondering if we should just not depend only on the host to provide xcode and setup brew and brew packages in our tasks code?

Reply:
The TF tasks don't use the cache as far as I know. They would have to be fixed. Would certainly be nicer.

Reply:
Indeed. I don't want to put more pressure on you, but since you have to re-do the workers, now is likely one of the best time. That being said, since I also plan to investigate how to get more performance out of them, maybe it can wait.